packages/core-p2p/src/utils/network-state.ts

```diff
@@ -1,70 +1,129 @@
-/* tslint:disable:no-shadowed-variable */
+/* tslint:disable:no-shadowed-variable member-ordering max-classes-per-file */
 
 import { app } from "@arkecosystem/core-container";
 import { slots } from "@arkecosystem/crypto";
 import { config as localConfig } from "../config";
+import { monitor } from "../monitor";
 
-const config = app.getConfig();
-
-/**
- * Returns current network state. Peers are update before the call
- * @param {Monitor} monitor
- * @private {Block} lastBlock
- * @returns {Object} JSON response for the forger to assess if allowed to forge or not
- */
-export = (monitor, lastBlock) => {
-    const createStateObject = (quorum, minimumNetworkReach, coldStart, overHeightBlockHeader) => ({
-        quorum,
-        nodeHeight: lastBlock.data.height,
-        lastBlockId: lastBlock.data.id,
-        overHeightBlockHeader,
-        minimumNetworkReach,
-        coldStart,
-    });
-
-    const peers = monitor.getPeers();
-    const minimumNetworkReach = localConfig.get("minimumNetworkReach", 20);
-    const currentSlot = slots.getSlotNumber();
-
-    let quorum = 0;
-    let noQuorum = 0;
-    let overHeightQuorum = 0;
-    let overHeightBlockHeader = null;
-
-    if (monitor.__isColdStartActive()) {
-        return createStateObject(0, true, true, overHeightBlockHeader);
+export class QuorumDetails {
```

Comment:
That is because the network state is only requested by the forger which does the logging:
https://github.com/ArkEcosystem/core/blob/develop/packages/core-forger/src/manager.ts#L246